src/containers/simulator/playPauseControl.js

```diff
@@ -1,13 +1,9 @@
 import React from 'react'
 import FontAwesome from 'react-fontawesome'
 
-
 const PlayPauseControl = ({ isRunning, handlePlay, handlePause }) => (
-  <div className="simulatorControl"  onClick={isRunning? handlePause : () => handlePlay(100)}>
-    {isRunning ?
-      <FontAwesome name="pause" size="2x" /> :
-      <FontAwesome name="play" size="2x" />
-    }
+  <div className="simulatorControl"  onClick={isRunning ? handlePause : handlePlay}>
```

Comment:
This is quite a nice improvement over the original dual output of FA icons, maybe it really was the killer feature!